src/masternodes/rpc_vault.cpp

```diff
@@ -141,6 +142,14 @@ namespace {
             collateralRatio = int(rate.val->ratio());
         }
 
+        bool isVaultTokenLocked {false};
+        for (const auto& collateral : collaterals->balances) {
```

Comment:
Vault will only lock on a loan token being split.

Reply:
Good call out. Should be harmless for now.